### ISSUE TITLE
[HC] The icon of "AddNew" in DataGridView control is unclear

### ISSUE THREAD

**Issue by MelonWang1** [dotnet/winforms#7722]
### .NET version

.NET 8.0.100-alpha.1.22451.15

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No, it is not a regression issue

### Issue description

**Current Behavior:**
The icon of "AddNew" in DataGridView control is unclear.
HC Aquatic theme:
<img width="306" alt="data-Aquatic" src="https://user-images.githubusercontent.com/94418985/188049625-0aa9ed3d-4986-4a3c-b6ab-a0cd460f6d16.png">
HC Dusk theme:
<img width="299" alt="data-Dusk" src="https://user-images.githubusercontent.com/94418985/188049641-ecb892c0-3016-4124-b375-ea6600d62f89.png">
HC Night Sky theme:
<img width="299" alt="data-Night sky" src="https://user-images.githubusercontent.com/94418985/188049656-ef39e018-0f78-414b-8340-6d90edf3a8f9.png">
**Expected Behavior:**
The icon of "AddNew" in DataGridView control should be clear.
win11 HC Desert theme:
<img width="300" alt="data-Desert" src="https://user-images.githubusercontent.com/94418985/188049787-ccacc7a1-ce56-4d2c-94eb-7c32497a14cc.png">
win10 HC White theme:
<img width="302" alt="data-HC White" src="https://user-images.githubusercontent.com/94418985/188049820-732cfb91-6ff8-4d15-b22e-d48e4fcbbd6c.png">

### Steps to reproduce

1. Turn on High Contrast theme in Windows settings
2. Create a WinForms .NET core project 
3. Add a DataGridView control
4. Expand smart tag and click "Add column..." to add a column
5. Ctrl+Shift+A add a Application Manifest File and uncomment windows 10
6. Build and run
7. Observe the icon of "AddNew"

**More info:**
1. This issue cannot repro on win11 HC Desert theme. 
2. This issue is repro on win10 machine but cannot repro on win10 HC White theme. 
3. This issue can repro on framework project.

**Comment by merriemcgaw** [comment]
Looks like the glyph needs to be filled in with white in those themes rather than just outlined in white.

**Comment by MelonWang1** [comment]
Verified on latest .NET 8.0.100-preview.2.23101.9, issue is fixed. Screenshot as below:
**Win10:**
HC #1 theme:
![HC_#1](https://user-images.githubusercontent.com/94418985/216273100-6045d2f5-a019-48f1-82c8-6872dab06720.png)

HC #2 theme:
![HC_#2](https://user-images.githubusercontent.com/94418985/216273129-e07088ec-7218-4f2d-861b-85a29f2adaed.png)

HC Black theme:
![HC_Black](https://user-images.githubusercontent.com/94418985/216273152-e96a5610-e010-4bf8-a8d8-fce3efad9cf9.png)

HC White theme:
![HC_White](https://user-images.githubusercontent.com/94418985/216273185-10afff05-67a9-4669-ae4d-54a700eb0d68.png)

**Win11:**
HC Aquatic theme:
![HC_Aquatic](https://user-images.githubusercontent.com/94418985/216273214-a1086d51-d63e-46f8-b999-141b44565e53.png)

HC Desert theme:
![HC_Desert](https://user-images.githubusercontent.com/94418985/216273229-70691647-2057-4fae-8f61-3b00e38cff82.png)

HC Dusk theme:
![HC_Dusk](https://user-images.githubusercontent.com/94418985/216273249-4dabd6cd-0ca0-4604-bce2-02bfa8ebd96a.png)

HC Night sky theme:
![HC_Night sky](https://user-images.githubusercontent.com/94418985/216273277-1478b89d-06f3-4d25-bdbb-7a18f51be01a.png)

**Comment by John-Qiao** [comment]
Verified with .NET 8 Preview 1 TP build: .NET 8.0.100-preview.1.23108.10, issue was fixed as same as above test result.
